### DISCUSSION TITLE
Socket Stress Example

### DISCUSSION THREAD

**Discussion by Deb-neal** [naver/ngrinder#878] [Q&A]
Socketio 서버 Stress 테스트 진행하려고 하는데

Artillery와는 다르게 Socket에 대한 정보가 없어 문의 드립니다.

혹시 Example이 있을까요??

더불어 이 글을  보시는 분들중에 Socket 서버 테스트 해보신분이 계시다면 

도움을 주시면 감사하겠습니다. 

P.S 여기에 올라와있는 글 중 하나가 Socket 관련 글이긴한데 2016년이라 너무 오래되어 적용하기엔 무리가 있더군요;

**Comment by donggyu04** [comment]
@Deb-neal 

안녕하세요.
현재 딱히 예제는 없습니다.
ngrinder 에서 기본적으로 제공하는 기능은 없고, [스크립트에서 외부 라이브러리 활용](https://github.com/naver/ngrinder/wiki/How-to-use-lib-and-resources) 을 참고하셔서, 해당 기능을 제공하는 외부 라이브러리가 있다면 사용하시는게 편하실 것 같습니다.

아니면 [과거가이드](https://github.com/naver/ngrinder/wiki/Using-nGrinder-to-perform-load-test-for-a-socket.io-app) 참고하셔서, 직접 클라이언트를 신규버전으로 구현하신 뒤 사용하셔야 할 것 같습니다.
